src/engine/channelhandle.h

```diff
@@ -116,6 +116,8 @@ inline uint qHash(const ChannelHandleAndGroup& handle_group) {
 class ChannelHandleFactory {
   public:
     ChannelHandleFactory() : m_iNextHandle(0) {
+        getOrCreateHandle("[Master]");
```

Comment:
Why is this required? Normally getOrCreateHandle can create the handle once it is used the first time.

Reply:
IIRC this was required to ensure both EngineMaster and effects were actually using the same handle, but I will look into the initialization process again and think about how it can be cleaned up.